doc/guides/kconfig/tips.rst

```diff
@@ -25,9 +25,13 @@ When deciding whether something belongs in Kconfig, it helps to distinguish
 between symbols that have prompts and symbols that don't.
 
 If a symbol has a prompt (e.g. ``bool "Enable foo"``), then the user can change
-the symbol's value in the ``menuconfig`` or ``guiconfig`` interface (or by
-manually editing configuration files). Therefore, only put a prompt on a symbol
-if it makes sense for the user to change its value.
+the symbol's value in the ``menuconfig`` or ``guiconfig`` interface (see
+:ref:`menuconfig`), or by manually editing configuration files. Therefore, only
+put a prompt on a symbol if it makes sense for the user to change its value.
```

Comment:
The second sentence can be dropped. This one  is much more useful:

> Conversely, a symbol without a prompt can never be changed by the user, not even by manually editing the configuration file.

This is dead simple, binary situation so it's more useful to describe it as such very plainly and very early rather than getting into what "makes sense" or not. There are countless things that make sense to some people and not to others.

Reply:
I've seen lots of lots of symbols in Zephyr and elsewhere that had prompts even though changing them didn't make sense, so I think it's helpful to put it like this, even if it might seem "obvious" in retrospect.

I think it's more subtle than it might seem, especially when starting out with Kconfig.

Reply:
OK, then I would just add my sentence and not remove anything.

Reply:
Added it. Checks that it looks alright in context.